index.html

```diff
@@ -1,22 +1,145 @@
+
+        <section class="midParagraph">The legendary players will meet against each other in what promises to be an epic
+          and unforgettable Go
+          match, representing the houses of Honinbo and Inoue, respectively.</section>
+
+        <section class="dateStyle LatoBold">1835.06.27(THU) ~ 28(FRI)</section>
+        <section class="locationStyle">@ Japan</section>
+      </section>
+    </div>
+
+    <main class="mainStyle LatoBold">
+      <h2 class="mainh2Style">Main Program</h2>
+
+      <div class="redbar"></div>
+
+      <section class="flexbox2">
+        <div class="flexbox1 divOpacity">
+          <img class="smallIcon" src="resources/program_icon_01.png" alt="Icon of Person at a Podium">
+          <div class="activityType">Seminar</div>
+          <div class="activityContent">The House of Honinbo Teaches: Life-and-Death Problems</div>
+        </div>
+
+        <div class="flexbox1 divOpacity">
+          <img class="smallIcon" src="resources/program_icon_02.png" alt="Presentation Blackboards">
+          <div class="activityType">Lecture</div>
+          <div class="activityContent">The House of Inoue Teaches: Playing Handicap</div>
+        </div>
+
+        <div class="flexbox1 divOpacity">
+          <img class="smallIcon" src="resources/program_icon_05.png" alt="Network Icon">
+          <div class="activityType">Exhibition</div>
+          <div class="activityContent">Do they play Go on other planets?</div>
+        </div>
+
+        <div class="flexbox1 divOpacity">
+          <img class="smallIcon" src="resources/program_icon_04.png" alt="Presentation Board">
+          <div class="activityType">Lecture</div>
+          <div class="activityContent">The Game of Go and Contemporary Physics</div>
+        </div>
 
-    <p>this upcoming June 27, 1835</p>
-    <h1>Honinbo Jowa plays Akaboshi Intetsu</h1>
-  </header>
+        <div class="flexbox1 divOpacity">
+          <img class="smallIcon" src="resources/program_icon_02.png" alt="Go Annotations Icon">
+          <div class="activityType">The Match</div>
+          <div class="activityContent">A Game of Go</div>
+        </div>
+      </section>
 
-  <main></main>
+      <div class="flexbox flexCenter">
+        <button class="buttonMainStyle">Attend this Year's Match</button>
+      </div>
 
-  <footer></footer>
+      <a class="programStyle">SEE THE WHOLE PROGRAM</a>
+    </main>
+
+    <section class="speakersSection"></section>
+
+    <p class="buttonMoreStyle">MORE
+      <img class="arrowStyle" src="resources/redArrowDown.svg" alt="Red Arrow Pointing Down">
+    </p>
+    </section>
+
+    <section class="LatoRegular styleSponsors">
+      <p class="pSponsors">Sponsors</p>
+
+      <p>The House of Honinbo</p>
+
+      <p class="Inoue">The House of Inoue</p>
+
+      <p>MIT</p>
+
+      <p class="ministry">The Japanese Ministry of Foreign Affairs</p>
+
+      <p>DuckDuckGo</p>
+    </section>
```

Comment:
Great design @fdezcaminero 🤝

- [ ] Kindly compare these two screenshots and then implement the changes accordingly. Your design should be a close match to the design given.
- Your design
![your footer](https://user-images.githubusercontent.com/63917725/235015713-b7348e1f-8051-4b37-ad98-2a777a1e8df0.PNG)
- Given design
![given footer](https://user-images.githubusercontent.com/63917725/235015754-7303556c-189d-410d-ab06-0cefe028cf8b.PNG)